.github/workflows/haskell.yml

```
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.6.4", "9.8.1"]
+        ghc: ["8.10.7", "9.6.4", "9.8.2", "9.10.1"]
```

do we need to pin the patch version?
```suggestion
        ghc: ["8.10.7", "9.6.4", "9.8.2", "9.10"]
```